src/app/app.component.ts

```diff
@@ -42,6 +43,7 @@ export class AppComponent {
   APP_VERSION = environment.version;
   DEPLOYMENT_NAME = environment.deploymentName;
   sideMenuDefaults = APP_SIDEMENU_DEFAULTS;
+  currentSkin: IAppSkin;
```

Comment:
We shouldn't need to access the current skin here, instead what we want is a better way of modifying and updating the app_constants in response to skin updates